common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -222,6 +230,24 @@ public boolean value(VirtualFile virtualFile) {
     ));
   }
 
+  // TODO: when is an Editor not an EditorImpl? why is this functionality not exposed in the intellij openapi Editor interface?
```

Comment:
you can browse through the subclasses of `Editor` to get an idea.

Reply:
This was meant to be a note to myself and has been removed. Thanks for the tip.